vectors/cryptography_vectors/x509/custom/no_sans.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
You can split out adding this vector (it needs to be documented in `test-vectors.rst` as well).